tests/by-util/test_tail.rs

```diff
@@ -363,7 +340,6 @@ fn test_follow_stdin_name_retry() {
 }
 
 #[test]
-#[cfg(target_os = "linux")]
```

Comment:
Is this test gonna work on all platforms eventually or can it only be fixed for some platforms?

Reply:
Good question...

Reply:
We should probably keep the `cfg` then, ut was probably added for a reason.

Reply:
ok